### PR TITLE
Handle decimal percentage on recurring donation

### PR DESCRIPTION
**Pull request by MohammadPCh**
- #3617

### CODE REVIEW

File: src/components/views/donate/RecurringDonationCard.tsx [hunk 1]
```diff
@@ -115,12 +122,15 @@ export const RecurringDonationCard = () => {
...
+					(
+						BigInt(_userStreamOnSelectedToken.currentFlowRate) *
 						ONE_MONTH_SECONDS *
-						100n) /
-					selectedToken.balance;
-				setPercentage(parseInt(_percentage.toString()));
+						100n
+					).toString(),
+				).dividedBy(selectedToken.balance.toString());
+				console.log('_percentage', _percentage.toString());
```
Comment (aminlatifi) [inline src/components/views/donate/RecurringDonationCard.tsx:132]: Please remove this

**Review by aminlatifi — APPROVED** [review]
Thanks @MohammadPCh 
Just a console.log to be removed.

**Review by alireza-sharifpour — APPROVED** [review]
Thanks @MohammadPCh

File: src/components/views/donate/RecurringDonationCard.tsx (hunk 1 above)
Comment (MohammadPCh) [reply src/components/views/donate/RecurringDonationCard.tsx]: Done

**Pull request merged**